verticapy/toolbox.py

```diff
@@ -443,6 +542,35 @@ def get_session(add_username: bool = True):
     return result
 
 
+# ---#
+def get_vertica_type(dtype):
+    if dtype in (str, "str", "string"):
+        dtype = "varchar"
+    elif dtype == float:
+        dtype = "float"
+    elif dtype == int:
+        dtype = "integer"
+    elif dtype == datetime.datetime:
+        dtype = "datetime"
+    elif dtype == datetime.date:
+        dtype = "date"
+    elif dtype == datetime.time:
+        dtype = "time"
+    elif dtype == datetime.timedelta:
+        dtype = "interval"
+    elif dtype == datetime.timezone:
+        dtype = "timestamptz"
+    elif dtype in (np.ndarray, np.array, list):
+        dtype = "array"
+    elif dtype == dict:
+        dtype = "row"
+    elif dtype == Union:
```

Comment:
should Union not have ""?

Reply:
right, but I don't know yet which dt the client will use so I will set it to tuple.